cpu/cc2538/dev/sys-ctrl.h

```diff
@@ -242,6 +242,36 @@
 #endif
 /** @} */
 /*---------------------------------------------------------------------------*/
+/** \name System clock divisor selection
+ * @{
+ */
+#ifdef SYS_CTRL_CONF_SYS_DIV
+#if SYS_CTRL_CONF_SYS_DIV & ~SYS_CTRL_CLOCK_CTRL_SYS_DIV
+#error Invalid system clock divisor
+#else
+#define SYS_CTRL_SYS_DIV        SYS_CTRL_CONF_SYS_DIV
+#endif
+#else
+#define SYS_CTRL_SYS_DIV        SYS_CTRL_CLOCK_CTRL_SYS_DIV_16MHZ
+#endif
+
+#ifdef SYS_CTRL_CONF_IO_DIV
+#if SYS_CTRL_CONF_IO_DIV & ~SYS_CTRL_CLOCK_CTRL_IO_DIV
+#error Invalid IO clock divisor
+#else
+#define SYS_CTRL_IO_DIV         SYS_CTRL_CONF_IO_DIV
+#endif
```

Comment:
Move this `#endif` to replace the `#else` above (this line can't be reached if the `#error` is triggered).